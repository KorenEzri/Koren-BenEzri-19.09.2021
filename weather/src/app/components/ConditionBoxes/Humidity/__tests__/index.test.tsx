import * as React from 'react';
import { render } from '@testing-library/react';

import { Humidity } from '..';


describe('<Humidity  />', () => {
  it('should match snapshot', () => {
    const loadingIndicator = render(<Humidity />);
    expect(loadingIndicator.container.firstChild).toMatchSnapshot();
  });
});
