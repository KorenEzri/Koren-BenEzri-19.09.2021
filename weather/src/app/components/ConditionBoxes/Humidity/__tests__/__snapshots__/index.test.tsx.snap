// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Humidity  /> should match snapshot 1`] = `
.c0 img {
  width: 150px;
  height: 150px;
  margin-left: 20px;
  margin-top: 10px;
}

.c0 span {
  position: relative;
  right: 90px;
  font-size: 26px;
  font-weight: bold;
}

<div
  class="c0"
>
  <img
    alt="humidity"
    src="humid.png"
  />
  <span />
</div>
`;
